phpcs.xml.dist

```diff
@@ -13,10 +13,17 @@
     <file>src</file>
     <file>docs/examples</file>
     <file>examples</file>
+    <file>generator/src</file>
+    <file>generator/config</file>
+    <file>generator/bin</file>
     <file>tests</file>
     <file>tools</file>
     <file>rector.php</file>
 
+    <!-- Ignore auto-generated code -->
+    <exclude-pattern>src/Builder/(Aggregation|Expression|Query|Stage)\.php</exclude-pattern>
+    <exclude-pattern>src/Builder/(Aggregation|Expression|Query|Stage)/*\.php</exclude-pattern>
+
     <!-- Target minimum supported PHP version -->
     <config name="php_version" value="70400"/>
```

Comment:
Should this also get a TODO item referencing [PHPLIB-1268](https://jira.mongodb.org/browse/PHPLIB-1268)?

Reply:
It is.